src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -1670,23 +1452,16 @@ class SaagieClient {
                     return JsonOutput.toJson(jobDetailResult)
                 }
             }
-        } catch (InvalidUserDataException invalidUserDataException) {
-            throw invalidUserDataException
-        } catch (GradleException stopActionException) {
-            throw stopActionException
-        } catch (Exception exception) {
-            logger.error('Unknown error in projectsCreate')
-            throw exception
-        }
+        }, 'Unknown error in projectsCreate');
```

Comment:
'Unknown error in task projectsCreate, function getListPipelineAndPipelineVersionsFromConfig'

Reply:
Log mistake